driver/CMakeLists.txt

```diff
@@ -1,36 +1,19 @@
 function(declare_odbc_lib_targets libname UNICODE)
 
-#option (UNICODE "Enable unicode" 0)
 if (UNICODE)
-    set(UNICODE 1)
-    if (ODBC_UNIXODBC)
-        option (SQL_WCHART_CONVERT "Unixodbc SQL_WCHART_CONVERT (use four bytes TCHAR)" 0)
-    endif ()
-    if(ODBC_IODBC)
-        #option (ODBC_CHAR16 "Strange option for iodbc" 1)
-    endif()
-    if (ODBC_IODBC AND APPLE)
-        set(ODBC_WCHAR 1)
-    endif()
-
-    set(_config_add w)
+    set(DRIVER_PREFIX Unicode)
+    set(WIDECHAR_SUFFIX w)
 else ()
-    set(UNICODE 0)
+    set(DRIVER_PREFIX ANSI)
+    set(WIDECHAR_SUFFIX)
 endif ()
 
-# Hell matrix
-#               TCHAR       sizeof()    note
-# not unicode   char        1
-# unicode:
-# win           wchar_t     2
-# iodbc LINUX   wchar_t     4           wchar_t != TCHAR
-# iodbc APPLE   wchar_t     4           wchar_t == TCHAR
-# unixodbc      wchar_t     4           true == SQL_WCHART_CONVERT
-# unixodbc      char16_t    2           false == SQL_WCHART_CONVERT
```

Comment:
too charismatic too simply get rid of that :)

Reply:
At first I wanted to keep those, but later decided to remove, as they provide insights into Driver Manager's implementation details, that could be ignored by driver author, given he uses proper abstractions in the driver code. Only 1-2 functions, that need to actually convert `SQLWCHAR` into internal encoding/representation, need to deal with these details. So here, this comment is rather misleading than useful (also it may not even cover all cases.)